build-system/tasks/dep-check.js

```diff
@@ -175,8 +175,13 @@ function getGraph(entryModule) {
 
   // TODO(erwinm): Try and work this in with `gulp build` so that
   // we're not running browserify twice on travis.
-  const bundler = browserify(entryModule, {debug: true, deps: true})
-      .transform(babelify, {compact: false});
+  const bundler = browserify(entryModule, {debug: true})
+      .transform(babelify, {
+        compact: false,
+        // Transform files in node_modules since deps use ES6 export.
```

Comment:
This is a significant change (especially before Babel 7), is this required?

Reply:
I believe so. I think being able to use ES module versions of dependencies is great consistency improvement, and may help `worker-dom` integration too. :)